prog/probe.go

```diff
@@ -46,25 +46,46 @@ var (
 	dockerEndpoint   = "unix:///var/run/docker.sock"
 )
 
-func check(flags map[string]string) {
```

Comment:
I introduced this here because our ebpf code made the cyclomatic complexity of probeMain go above the threshold, but yes, it's an unrelated change. Here's the PR for that: https://github.com/weaveworks/scope/pull/2148